code/modules/shuttle/on_move.dm

```diff
@@ -16,13 +16,13 @@
     // Stationary ports shouldn't move, mobile ones move themselves
     return 0
 
-/obj/machinery/door/onShuttleMove()
+/obj/machinery/door/airlock/onShuttleMove()
```

Comment:
Amusingly without the `airlock` bit, it would open any child of `obj/machinery/door`, including stuff like wooden doors 🤷‍♂️

Reply:
This is fine.